srcpkgs/akregator/template

```diff
@@ -11,6 +11,6 @@ makedepends="knotifyconfig-devel kontactinterface-devel ktexteditor-devel
 short_desc="KDE feed reader"
 maintainer="Helmut Pozimski <helmut@pozimski.eu>"
 license="GPL-2.0-or-later, GFDL-1.2-only"
-homepage="https://www.kde.org/applications/internet/akregator"
-distfiles="${KDE_SITE}/applications/${version}/src/${pkgname}-${version}.tar.xz"
-checksum=c3aab6ebba25eab8c76b113a900af33a90222257f957650d0bbc0c47d5cf4bc9
+homepage="https://www.kde.org/release-service/internet/akregator"
```

Comment:
homepage shouldn't be changed

Reply:
Indeed, seems my sed command was a bit to naive. I'm going to correct that.